src/api/app/views/webui/user/show.html.haml

```diff
@@ -0,0 +1,170 @@
+- @pagetitle = "Home of #{@displayed_user}"
```

Comment:
this file is huge, we should split it (not in this PR but we should do it) :confounded: